pkgs/development/python-modules/srsly/default.nix

```diff
@@ -12,17 +13,19 @@
 
 buildPythonPackage rec {
   pname = "srsly";
-  version = "2.3.2";
+  version = "2.4.0";
 
   disabled = pythonOlder "3.6";
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "f78eaef8d982e98921ce026d4205e652a9333137259f0b621f5c7b579e746e9d";
+    sha256 = "e29730be53015970e4a59050e8e9f9be44d762108a617df56c9dfc981b515ab7";
   };
 
   nativeBuildInputs = [ cython ];
 
+  propagatedBuildInputs = [ catalogue ];
+
   checkInputs = [
     mock
     numpy
```

Comment:
Can you check if the tests are working now? Otherwise we might want to remove the checkInputs.

Reply:
Unfortunately, they still don't :(.

I did check the test dependencies, and they are still up to date. In the past reviewers have told me to keep dependencies around in such cases as documentation.

Reply:
If they are still correct we can just leave them in.

Reply:
The tests fails because they've vendored ruamel_yaml and the cyaml test cannot find it? Odd.